js_modules/dagit/packages/core/src/instance/BackfillTable.mocks.ts

```diff
@@ -0,0 +1,419 @@
+import {MockedResponse} from '@apollo/client/testing';
+
+import {BulkActionStatus, RunStatus} from '../graphql/types';
+
+import {
+  SINGLE_BACKFILL_STATUS_COUNTS_QUERY,
+  SINGLE_BACKFILL_STATUS_DETAILS_QUERY,
+} from './BackfillRow';
+import {SingleBackfillCountsQuery, SingleBackfillQuery} from './types/BackfillRow.types';
+import {BackfillTableFragment} from './types/BackfillTable.types';
+
+function buildTimePartitionNames(start: Date, count: number) {
+  const results: string[] = [];
+  for (let ii = 0; ii < count; ii++) {
+    start.setMinutes(start.getMinutes() + 15);
+    results.push(start.toISOString());
+  }
+  return results;
+}
+
+export const BackfillTableFragmentRequested2000AssetsPure: BackfillTableFragment = {
+  backfillId: 'qtpussca',
+  status: BulkActionStatus.REQUESTED,
+  numPartitions: 2000,
+  timestamp: 1675196684.587724,
+  partitionSetName: null,
+  partitionSet: null,
+  error: null,
+  numCancelable: 0,
+  partitionNames: buildTimePartitionNames(new Date('2020-01-01'), 2000),
+  assetSelection: [
+    {
+      path: ['global_graph_asset'],
+      __typename: 'AssetKey',
+    },
+  ],
+  __typename: 'PartitionBackfill',
+};
+
+export const BackfillTableFragmentRequested2000AssetsPureStatus: MockedResponse<SingleBackfillCountsQuery> = {
```

Comment:
Very longwinded name here, but was thinking these mocks should have some sort of consistent naming...  "fragment name + some descriptive info about this test case"?

Reply:
+1